test/Interop/Cxx/members/Inputs/method-calls-method-from-nested-struct.h

```diff
@@ -0,0 +1,15 @@
+#ifndef TEST_INTEROP_CXX_MEMBERS_INPUTS_METHOD_CALLS_METHOD_FROM_NESTED_STRUCT_H
+#define TEST_INTEROP_CXX_MEMBERS_INPUTS_METHOD_CALLS_METHOD_FROM_NESTED_STRUCT_H
+
+struct IncrementUser {
+  struct Incrementor {
```

Comment:
What about a test for a nested struct that calls an inline function in its constructor?

Reply:
Done.